src/screens/Damages/index.jsx

```diff
@@ -47,8 +51,78 @@ const styles = StyleSheet.create({
   avatar: {
     marginHorizontal: spacing(2),
   },
+  floatingButton: {
+    position: 'absolute',
+    bottom: 0,
+    right: 0,
+  },
+  validationButton: { margin: spacing(2) },
+  buttonLabel: { color: '#FFFFFF' },
+  dialog: { maxWidth: 450, alignSelf: 'center', padding: 12 },
+  dialogDrawing: { display: 'flex', alignItems: 'center' },
+  dialogContent: { textAlign: 'center' },
+  dialogActions: { width: '100%' },
 });
 
+function DialogModal({ isDialogOpen, handleDismissDialog }) {
+  const route = useRoute();
+  const { inspectionId } = route.params;
+  // we need error handling here
+  const { isLoading, request } = useRequest(
+    updateOneTaskOfInspection({
+      inspectionId,
+      taskName: 'damage_detection',
+      data: { status: taskStatuses.VALIDATED },
+    }),
+    { onSuccess: handleDismissDialog },
+    false,
+  );
+  return (
+    <Portal>
+      <Dialog
+        visible={Boolean(isDialogOpen)}
+        onDismiss={handleDismissDialog}
+        style={styles.dialog}
+      >
+        <View style={styles.dialogDrawing}>
+          <Drawing xml={submitDrawing} width="200" height="120" />
+        </View>
+        <Dialog.Title style={styles.dialogContent}>
+          Are you sure?
+        </Dialog.Title>
+
+        <Dialog.Content>
+          <Paragraph style={styles.dialogContent}>
+            Please confirm your damage validation
+          </Paragraph>
+        </Dialog.Content>
+
+        <Dialog.Actions>
```

Comment:
Can you put the buttons into one CardActions et style the container ? You can see other modifications I've made on dialogs.